### DISCUSSION TITLE
feat(AutoComplete): DisplayCount Parameter only working when the input gets Onfocus

### DISCUSSION THREAD

**Discussion by mrtonyguo** [dotnetcore/BootstrapBlazor#3903] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

Steps :
复现步骤:

1. Setting the Parameter of AutoComplete.razor, DisplayCount = "20";

   **设置自动完成框显示的数据数量, 20或者其他**；

![image](https://github.com/user-attachments/assets/4070b7cc-ce77-4578-a2e7-e269570e54f2)

3. If setting the OnFocusFilter = "true", the DisplayCount will be working;

   **如果设置了 OnFocusFilter 为true，下拉框显示的数据数量正常;**

5. The default OnFocusFilter is false, if no setting of OnFocusFilter in the AutoComplate.razor, the whole Items will be shown, the DisplayCount function is not working;

   **OnFocusFilter的默认值为False，如果AutoComplete的没有设置OnFocusFilter为True，下拉框会显示所有的数据，DisplayCount功能没生效;**

Tips:
The AutoComplate.razor used the @key to enhance the Rendering Performance, suggest for the @bind-Value List<string> must be disinct.

**自动完成框组件使用了@key提高渲染性能，绑定的字符串列表必须是不重复的字符，可以使用DISTINCT”，用于返回唯一不重复的结果集;**

### Describe the solution you'd like

Add the condition judgement to the OnFocus Method:

**是否可以在OnFocus的方法内加上对于DisplayCount的一个判定;**

![$5ANF27 UFD_2VDFFPFVVU](https://github.com/user-attachments/assets/a39ff466-96c4-40a3-a2ff-959abfdec888)

**Comment by bb-auto** [comment]
@mrtonyguo Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply, marked as answer]
@mrtonyguo 发布新版本 `8.7.4-beta04`
